apluslms_roman/builder.py

```diff
@@ -10,18 +10,24 @@
 
 class Builder:
     def __init__(self, engine, config, observer=None):
-        assert isdir(config.__path__), "Course configuration path doesn't exists"
+        if not isdir(config.dir):
+            raise ValueError("config.dir isn't a directory.")
         self.config = config
-        self.path = config.__path__
+        self.path = config.dir
         self._engine = engine
         self._observer = observer or StreamObserver()
 
-    def build(self):
+    def buildSteps(self):
```

Comment:
Typically we use underscores here (`build_steps`). In addition, as this is "getter", we can use property again, though, you could add `step_list` / `step_refs` as parameter. Thus, this probably should be `def get_steps(refs)`.